src/renderer/pages/modelling-ui-poc/column-group/column-group.tsx

```diff
@@ -0,0 +1,171 @@
+import { useEffect } from 'react';
+import { WebGLRenderer, OrthographicCamera, Scene, Color, Group } from 'three';
+import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls';
+import invariant from 'tiny-invariant';
+import { InfiniteGrid, ModelGrid, Column } from 'renderer/three/components';
+import { CSS3DRenderer } from 'three/examples/jsm/renderers/CSS3DRenderer';
+import { COLORS } from 'renderer/constants';
+
+const ColumnSystem = () => {
```

Comment:
can you rename the component to match the file name?

Reply:
sure!